src/withMockTime.ts

```diff
@@ -2,15 +2,15 @@ import type {
   Renderer,
   PartialStoryFn,
   StoryContext,
-} from "@storybook/types";
+} from "storybook/internal/types";
 import FakeTimers from "@sinonjs/fake-timers";
 
 const now = new Date();
 let clock: FakeTimers.Clock | undefined;
 
 export const withMockTime = (
   StoryFn: PartialStoryFn<Renderer>,
-  context: StoryContext<Renderer>
+  context: StoryContext<Renderer>,
```

Comment:
My IDE's default linter... 😮‍💨 let me know if you want me to clean those up or add a basic linter setup.